public/index.html

```diff
@@ -0,0 +1,215 @@
+          <li class="mapLi"><button class="mapButtons" data-content="august" id="august" type="button" name="button">august</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="september" id="september" type="button" name="button">september</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="october" id="october" type="button" name="button">october</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="november" id="november" type="button" name="button">november</button></li>
+          <li class="mapLi"><button class="mapButtons" data-content="december"id="december" type="button" name="button">december</button></li>
+        </ul>
+
+
+      <section id="mapBox" style="background-image: url({{map}})">
+        <canvas id="coffeeData" width="300" height="525"></canvas>
+      </section>
+      </section>
+    </section>
+    <section id="brewHead">
+      <h1 id="brewH1">How we Brew it</h1>
+      <img id ="straightArrowLeft" class="arrows" src="/media/images/straightArrowLeft.png" alt="arrow" />
+    </section>
+    <section class="coffeePages" id="brewMethods">
+      <section id="methodsContainer">
+        <section class="methodEls" id="brewImage">
+          <img src="{{brewPic}}" alt="not found" />
+        </section>
+        <section class="methodEls" id="brewSteps">{{brewInstructions}}</section>
+      </section>
+    </section>
+  </section>
+  </script>
+
+
+
+  </head>
+  <body>
+    <select id="coffeeCountry" name="Country">
+      <option class="menuOptions" value="nicaraguaPage.json">Nicaragua</option>
+      <option class="menuOptions" value="hondurasPage.json">Honduras</option>
+      <option class="menuOptions" value="guatemalaPage.json">Guatemala</option>
+
+    </select>
+    <header class="tab header">
+      <a href="/">
+      <section class="logo">
+        <h1 id="origin1H1">origin1<span id="cs"><br>coffee + source</br></span></h1>
+      </section>
+      <div class="navCollapsed">
+        <div id="navButton">
+        </a>
+          <img src="media/images/arabica-beanUse.png"/>
+        </div>
+        <nav>
+          <ul>
+            <li class="tab nav" data-content="home"><a href="/">Home</a></li>
+            <li class="tab nav" data-content="coffee"><a href="/coffee">Coffee</a></li>
+            <li class="tab nav" data-content="contact"><a href="/contact">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section class="homePage tab">
+      <section class="videoContainer">
+
+        <video class="bgVid tab" src="media/videos/vidTest.mp4" autoplay></video>
+      </section>
+      <section class="homeBodyBox tab">
+        <section class="logo ">
+          <h1 id="origin1H1">origin1<span id="cs"><br>coffee + source</br></span></h1>
+        </section>
+        <section id="bean">
+          <img id="beanImg" src="media/images/arabica-beanUse.png" alt="Not Found" />
```

Comment:
Quick note on the `alt='Not Found'`. What happens if someone is using a screen reading, and the photo is rendering but the reader is providing the user with the alt text for the image? 

`alt` should be an alternative text format describing what your image is displaying. You can also use this as an additional way to provide relevance / SEO to images on your site.